any/barcode-gas/barcode-gas-componentsize.md

```diff
@@ -0,0 +1,212 @@
+# Return component size from barcode
+
+This example shows how to extract the component size or sizes for a given gas distribution components encoded using the [ASTM F2897 standard](https://www.astm.org/Standards/F2897.htm). This specification defines requirements for the data used in the tracking and traceability base-62 encoding system and the format of the resultant code to characterize various components used in fuel gas piping systems.
```

Comment:
I would remove the `a` in the first sentence so it looks like: `This example shows how to extract the component size or sizes for given gas distribution components encoded using the ASTM F2897 standard.`

This applies to the first sentence of all the descriptions since they follow the same format.

Reply:
Made the change for all descriptions.